main.nf

```diff
@@ -115,12 +115,12 @@ params.bed12 = params.genome ? params.genomes[ params.genome ].bed12 ?: false :
 params.hisat2_index = params.genome ? params.genomes[ params.genome ].hisat2 ?: false : false
 
 
-ch_mdsplot_header = Channel.fromPath(file("$baseDir/assets/mdsplot_header.txt"))
-ch_heatmap_header = Channel.fromPath(file("$baseDir/assets/heatmap_header.txt"))
-ch_biotypes_header = Channel.fromPath(file("$baseDir/assets/biotypes_header.txt"))
-ch_multiqc_config = Channel.fromPath(file(params.multiqc_config))
-ch_output_docs = Channel.fromPath(file("$baseDir/docs/output.md"))
-Channel.fromPath(file("$baseDir/assets/where_are_my_files.txt"))
+ch_mdsplot_header = Channel.fromPath("$baseDir/assets/mdsplot_header.txt")
+ch_heatmap_header = Channel.fromPath("$baseDir/assets/heatmap_header.txt")
+ch_biotypes_header = Channel.fromPath("$baseDir/assets/biotypes_header.txt")
+ch_multiqc_config = Channel.fromPath(params.multiqc_config))
```

Comment:
https://github.com/nf-core/rnaseq/blob/b71a891533d68d076c28647aeb3847ad565a7c6d/main.nf#L161

Reply:
This should also be put in a channel, right?